@xen-orchestra/lite/src/components/pool/network/PoolNetworksSidePanel.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <UiPanel>
+    <template v-if="network" #header>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        variant="tertiary"
+        size="medium"
+        accent="info"
+        :left-icon="faEdit"
+      >
+        {{ $t('edit') }}
+      </UiButton>
+      <UiButton
+        v-tooltip="$t('coming-soon')"
+        disabled
+        variant="tertiary"
+        size="medium"
+        accent="danger"
+        :left-icon="faTrash"
+      >
+        {{ $t('delete') }}
+      </UiButton>
+      <UiButtonIcon v-tooltip="$t('coming-soon')" disabled accent="info" size="medium" :icon="faEllipsis" />
+    </template>
+    <template #default>
+      <VtsNoSelectionHero v-if="!network" type="panel" />
+      <template v-else>
+        <UiCard class="card-container">
+          <UiCardTitle v-tooltip="{ placement: 'bottom-end' }" class="typo p1-medium text-ellipsis">
+            {{ network.name_label }}
+          </UiCardTitle>
+          <div class="content">
+            <!-- UUID -->
+            <VtsCardRowKeyValue>
+              <template #key>
+                {{ $t('uuid') }}
+              </template>
+              <template #value>{{ formatValue(network.uuid) }}</template>
+              <template #addons>
+                <UiButtonIcon
+                  v-if="network.uuid"
```

Comment:
It would be preferable to move the condition on the `template`.

Otherwise, the condition on `<div v-if="slots.addons" class="addons">` inside `VtsCardRowKeyValue.vue` will always be `true`, even if the `UiButtonIcon` is removed.

```suggestion
              <template #addons v-if="network.uuid">
                <UiButtonIcon
```